### PR TITLE
Sdk node with fetch

### PR DESCRIPTION
**Pull request by SokratisVidros**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [x] `@clerk/nextjs`
- [x] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/clerk-expo`
- [x] `@clerk/backend-core`
- [x] `@clerk/clerk-sdk-node`
- [x] `@clerk/edge`
- [ ] `build/tooling/chore`

## Description

- [ ] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

The purpose of this PR is to refactor Clerk Backend SDKs in order to make them cross-platform for Node and V8 runtimes. This first PR was inspired by https://www.edgedb.com/blog/how-we-converted-our-node-js-library-to-deno-using-deno.

### CODE REVIEW

File: packages/sdk-node/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "package.json"
   ],
   "engines": {
-    "node": ">=12"
+    "node": ">=16"
```
[the remark this answers is not in the document]
Comment (SokratisVidros) [reply packages/sdk-node/package.json:15]: You are right. This change should be reverted.

[the remark this answers is not in the document]
Comment (SokratisVidros) [reply packages/sdk-node/package.json]: Reverted.

**Review by gkats — APPROVED** [review]

**Review by igneel64 — APPROVED** [review]

File: packages/backend-core/API.md [hunk 2]
```diff
@@ -382,6 +390,20 @@ const membership = await clerkAPI.organizations.deleteOrganizationMembership({
...
+
+#### createRedirectUrl({ url })
+
+Creates a new redirect URL:
+
+```
+const redirectUrl = await clerkAPI.redirectUrls.createRedirectUrl({ url });
+```
+
+TODO
```
[the remark this answers is not in the document]
Comment (igneel64) [reply packages/backend-core/API.md]: Done!

File: packages/backend-core/API.md [hunk 3]
```diff
@@ -434,6 +456,10 @@ const sessionToken = 'my-session-token';
 const session = await clerkAPI.sessions.verifySession(sessionId, sessionToken);
 ```
 
+## Sign in token operations
+
+TODO
```
[the remark this answers is not in the document]
Comment (igneel64) [reply packages/backend-core/API.md]: Done :)

**Pull request merged**